### ISSUE TITLE
Legacy Branch: uninitialized constant Dor::Workflow::Service (NameError)

### ISSUE THREAD

**Issue by blalbrit** [sul-dlss/pre-assembly#470]
```
[lyberadmin@sul-lyberservices-prod current]$ ROBOT_ENVIRONMENT=production bundle exec bin/pre-assemble /dor/staging/GBTEST/gb_test.yaml
Traceback (most recent call last):
	14: from bin/pre-assemble:3:in `<main>'
	13: from bin/pre-assemble:3:in `require'
	12: from /home/lyberadmin/pre-assembly/releases/20190408203624/config/boot.rb:17:in `<top (required)>'
	11: from /home/lyberadmin/pre-assembly/shared/bundle/ruby/2.5.0/gems/confstruct-0.2.7/lib/confstruct/hash_with_struct_access.rb:189:in `method_missing'
	10: from /home/lyberadmin/pre-assembly/shared/bundle/ruby/2.5.0/gems/confstruct-0.2.7/lib/confstruct/hash_with_struct_access.rb:85:in `[]'
	 9: from /usr/local/rvm/rubies/ruby-2.5.3/lib/ruby/2.5.0/delegate.rb:349:in `block in delegating_block'
	 8: from /home/lyberadmin/pre-assembly/shared/bundle/ruby/2.5.0/gems/confstruct-0.2.7/lib/confstruct/utils.rb:7:in `eval_or_yield'
	 7: from /home/lyberadmin/pre-assembly/shared/bundle/ruby/2.5.0/gems/dor-services-5.32.1/lib/dor/config.rb:84:in `block (2 levels) in <class:Configuration>'
	 6: from /home/lyberadmin/pre-assembly/shared/bundle/ruby/2.5.0/gems/dor-workflow-service-2.12.0/lib/dor/services/workflow_service.rb:570:in `configure'
	 5: from /home/lyberadmin/pre-assembly/shared/bundle/ruby/2.5.0/gems/dor-workflow-service-2.12.0/lib/dor/services/workflow_service.rb:584:in `build_connection'
	 4: from /home/lyberadmin/pre-assembly/shared/bundle/ruby/2.5.0/gems/faraday-0.15.4/lib/faraday.rb:70:in `new'
	 3: from /home/lyberadmin/pre-assembly/shared/bundle/ruby/2.5.0/gems/faraday-0.15.4/lib/faraday.rb:70:in `new'
	 2: from /home/lyberadmin/pre-assembly/shared/bundle/ruby/2.5.0/gems/faraday-0.15.4/lib/faraday/connection.rb:89:in `initialize'
	 1: from /home/lyberadmin/pre-assembly/shared/bundle/ruby/2.5.0/gems/dor-workflow-service-2.12.0/lib/dor/services/workflow_service.rb:593:in `block in build_connection'
/home/lyberadmin/pre-assembly/shared/bundle/ruby/2.5.0/gems/dor-workflow-service-2.12.0/lib/dor/services/workflow_service.rb:598:in `user_agent': uninitialized constant Dor::Workflow::Service (NameError)
```

Per @jcoyne - looks like dependencies were updated but not tested on the legacy branch.

**Comment by blalbrit** [comment]
This makes it impossible to do the more granular work needed by the SDR managers (and would be great if we ported that functionality into the app and retired the two-branch approach)

**Comment by vivnwong** [comment]
The work to retire the two-branch approach is already scheduled after the team finishes the Sinopia work cycle. 
For this specific problem, should we have someone to investigate this?

**Comment by peetucket** [comment]
making some attempts here: #471 

so far just tried updating dor-services gem, didn't work locally

**Comment by peetucket** [comment]
fixed by #472
